apps/hash-ai-worker-ts/src/activities/flow-activities/research-entities-action/open-ai-coordinating-agent.ts

```diff
@@ -181,38 +184,48 @@ const getNextToolCalls = async (params: {
     );
   }
 
-  const { message, usage: _usage } = llmResponse;
+  const { message } = llmResponse;
 
   const toolCalls = getToolCallsFromLlmAssistantMessage({ message });
 
-  /** @todo: capture usage */
-
   return { toolCalls };
 };
+
 const createInitialPlan = async (params: {
   input: CoordinatingAgentInput;
+  questionsAndAnswers?: string;
 }): Promise<{ plan: string }> => {
-  const { input } = params;
+  const { input, questionsAndAnswers } = params;
 
   const systemPrompt = dedent(`
     ${generateSystemPromptPrefix({ input })}
-
-    Do not make *any* tool calls. You must first provide a plan of how you will use
-      the tools to progress towards completing the task.
-
-    This should be a list of steps in plain English.
+    
+    ${questionsAndAnswers ? `You previously asked the user clarifying questions on the research brief, and received the following answers:\n${questionsAndAnswers}` : ""}
```

Comment:
The question answer pairs are only going to be visible in the system prompt when creating the plan, is this intentional?

I would've expected that the question answer pairs get added to the `state` of the coordinator agent, and then used in the `generateSystemPromptPrefix` method so that they're in context when creating the initial plan, and when the agent is making subsequent tool calls.